base/src/main/java/io/spine/code/proto/EnumType.java

```diff
@@ -46,7 +47,8 @@ private EnumType(EnumDescriptor descriptor,
         super(descriptor, descriptorProto, className, typeUrl);
     }
 
-    private static EnumType create(EnumDescriptor descriptor) {
+    @VisibleForTesting
+    public static EnumType create(EnumDescriptor descriptor) {
```

Comment:
Please find a way to make this method package-private. `public` is way too broad.